packages/pelagos/src/listInput/ListEntries.js

```diff
@@ -52,26 +65,36 @@ const ListEntries = ({
 		return clearHighlight && clearHighlight.cancel;
 	}, [highlightKey, clearHighlight]);
 
+	const operationId = `${id}-operation`;
 	return (
 		<>
 			<div className="sr-only" aria-live="polite" ref={liveRef} />
+			{reorderable && (
+				<div id={operationId} className="sr-only">
+					{t`Press space bar to reorder`}
+				</div>
+			)}
 			<ul
+				ref={reorderRef}
 				id={id}
-				className={`ListEntries ListEntries--${column ? 'column' : 'grid'}${className ? ` ${className}` : ''}`}
+				className={`ListEntries ListEntries--${column || reorderable ? 'column' : 'grid'}${className ? ` ${className}` : ''}`}
 				onClick={handleClick}>
 				{list.map((item, i) => {
 					const name = getItemName(item);
 					const element = renderItem ? renderItem(item) : renderListItem(name);
-					let className = element.props.className;
-					className = className ? `${className} ListEntries__name` : 'ListEntries__name';
 					const itemKey = getItemKey(item, i);
 					return (
 						<Layer
 							key={itemKey}
 							as="li"
 							className={`ListEntries__item${itemKey === highlightKey ? ' ListEntries__item--highlight' : ''}`}
+							tabIndex={reorderable ? 0 : undefined}
+							aria-describedby={reorderable ? operationId : undefined}
 							data-testid="list-item">
-							{cloneElement(element, {className})}
+							<div className={`ListEntries__name${reorderable ? ' draggable' : ''}`}>
```

Comment:
Previously, any supplied `className` would end up in the `className` of these list items. That doesn't seem like the expected behaviour to me, but maybe there is a reason behind it. Will need to wait for @jcestibariz to look over this.

Reply:
This isn't the `className` supplied to `ListEntries`, but the constructed classname of `ListEntries__name` + whatever class `renderItem` has. I believe the `cloneElement` here was to inject `ListEntries__name` class to the list items

Reply:
Ah I see, I missed `let className = element.props.className`